pool_automation/roles/aws_manage/defaults/main.yml

```diff
@@ -1,11 +1,13 @@
 ---
-# AWS primary parameters
-aws_ec2_type: t2.micro
+# General
+aws_project_name: PoolAutomation
+aws_project_name_short: PA
```

Comment:
Why two separate names? Feels like it could lead to confusion at some point.

Reply:
Renamed the latter. We need full name and some short form for prefixes.